packages/2018-local-elections/src/components/OutraisingYourOpponent/index.js

```diff
@@ -88,6 +88,7 @@ export class OutraisingYourOpponent extends React.Component {
           sortOrder="sortOrder"
           data={data}
           dataValueFormatter={percentage}
+          domain={{x: [0,0.6], y: [1,11]}}
```

Comment:
This is very specific and not associated with the data...but I'll turn a blind eye 🙈

Reply:
🙈